backend/pkg/types/cis_docker_benchmark.go

```diff
@@ -40,6 +41,7 @@ func cisDockerBenchmarkResultFromBackendAPI(in *models.CISDockerBenchmarkCodeInf
 	return &CISDockerBenchmarkResult{
 		Code:         in.Code,
 		Level:        in.Level,
+		Titles:       getTitlesFromBackendAPI(in.Assessments),
```

Comment:
not sure i understand. a specific code can have multiple titles and descriptions?

Reply:
Maybe it's better to get `title` based on the code.